README.md

```diff
@@ -46,5 +46,5 @@ $ disease out.hex
    4:   ADD
    5:   POP
 ```
-
-
+### Dependencies
+`ecfg` requires z3 to build
```

Comment:
Would you mind adding an example of installing it, on whatever platform you use?

Ubuntu might look something like:

```
sudo apt install z3 libz3-dev
```

Reply:
Sure, let me know if this update works!